python/dgl/transform.py

```diff
@@ -1393,56 +1393,159 @@ def out_subgraph(g, nodes):
         ret.edges[etype].data[EID] = induced_edges[i].tousertensor()
     return ret
 
-def to_simple(g, return_counts='count', writeback_mapping=None):
-    """Convert a heterogeneous multigraph to a heterogeneous simple graph, coalescing
-    duplicate edges into one.
+def to_simple(g, return_counts='count', writeback_mapping=False, copy_ndata=True, copy_edata=False):
+    r"""Convert a graph to a simple graph without duplicate edges.
 
-    This function does not preserve node and edge features.
+    For a heterograph with multiple edge types, we can
+    treat edges corresponding
+    to each type as a separate graph and convert each
+    of them to a simple graph.
+
+    Writeback mappings can be returned for each edge
+    type subgraph. For the edges in the original graph,
```

Comment:
When ``writeback_mapping=True``, an extra mapping is returned. For the edges ...